langchain/src/llms/openai.ts

```diff
@@ -302,10 +308,9 @@ export class OpenAI extends BaseLLM implements OpenAIInput {
                       choice.finish_reason = part.finish_reason;
                       choice.logprobs = part.logprobs;
                       // eslint-disable-next-line no-void
-                      void this.callbackManager.handleLLMNewToken(
-                        part.text ?? "",
-                        true
-                      );
+                      void this.configureCallbackManager(
```

Comment:
So we create a new callback manager for every token logged? That's a lot of callback managers

Reply:
yep, can fix